src/libraries/System.Private.CoreLib/src/System/BitConverter.cs

```diff
@@ -183,6 +183,25 @@ public static bool TryWriteBytes(Span<byte> destination, ulong value)
             return true;
         }
 
+        // Converts a half into an array of bytes with length
+        // two.
+        public static unsafe byte[] GetBytes(Half value)
+        {
+            byte[] bytes = new byte[sizeof(Half)];
+            Unsafe.As<byte, Half>(ref bytes[0]) = value;
```

Comment:
```suggestion
            Unsafe.As<byte, Half>(ref MemoryMarshal.GetArrayDataReference(bytes)) = value;
```
Can this be used here?

Reply:
> I didn't check if the JIT already eliminates the bound check in the current code.

It does.